src/test/java/org/broadinstitute/hellbender/utils/mcmc/GibbsSamplerCopyRatioUnitTest.java

```diff
@@ -0,0 +1,412 @@
+package org.broadinstitute.hellbender.utils.mcmc;
```

Comment:
I would say that the data should be generated on the fly and not loaded, but for the future integration test that I'm assuming loading data seems more right, so it makes sense to keep it as it is here.